docker/Makefile

```diff
@@ -26,7 +26,7 @@ up: down
 .PHONY: up
 
 down:
-	LOTUS_IMAGE_TAG=v1.2.2 \
+	LOTUS_IMAGE_TAG=v1.4.0 \
```

Comment:
Same here, not sure why my merge of master is resulting in this, but I think it's right.